lhotse/dataset/speech_synthesis.py

```diff
@@ -15,38 +15,54 @@ class SpeechSynthesisDataset(torch.utils.data.Dataset):
     .. code-block::
 
         {
-            'audio': (B x NumSamples) tensor
-            'features': (B x NumFrames x NumFeatures) tensor
-            'tokens': list of lists of characters (str)
+            'audio': (B x NumSamples) float tensor
+            'features': (B x NumFrames x NumFeatures) float tensor
+            'tokens': (B x NumTokens) int tensor
+            'features_lens': (B x NumFrames) int tensor
+            'tokens_lens': (B x NumTokens) int tensor
         }
     """
 
-    def __init__(self, cuts: CutSet) -> None:
+    def __init__(
+        self,
+        cuts: CutSet,
+        feature_transforms: Union[Iterable[Callable], Callable] = [],
```

Comment:
mutable default arguments are dangerous; it is better to write `= None` and handle the case with `if x is None: self.x = []`

Reply:
Should it then be typed with Optional? `Optional[Sequence[Callable]] = None`?

Reply:
I think it's going to become difficult to read. I'm OK to omit Optional in such cases.